lib/pengine/unpack.c

```diff
@@ -1375,6 +1375,47 @@ unpack_status(xmlNode * status, pe_working_set_t * data_set)
     return TRUE;
 }
 
+/*!
+ * \internal
+ * \brief Unpack node's time when it became a member at the cluster layer
+ *
+ * \param[in]     node_state  Node's node_state entry
+ * \param[in,out] data_set    Cluster working set
+ *
+ * \return Epoch time when node became a cluster member
+ *         (or 1 for legacy entries) if a member, 0 if not a member,
+ *         or -1 if no valid information available
+ */
+static long long
+unpack_node_member(const xmlNode *node_state, pe_working_set_t *data_set)
+{
+    const char *member_time = crm_element_value(node_state, PCMK__XA_IN_CCM);
+    int member = 0;
+
+    if (member_time == NULL) {
```

Comment:
Yes. Only remote nodes should default it to true. Less than ideal, but needed for backward compatibility with entries recorded by controllers <=1.1.14. If we drop that compatibility, we can change `remote_state_from_cib()` rather than here.